### PR TITLE
Adjust EVM documentation for TestNet deployment.

### PR DESCRIPTION
**Pull request by frankbraun**
The EVM documentation has to be adjusted from BetaNet to TestNet.

### CODE REVIEW

**Comment by frankbraun** [comment]
Depublished a few pages which need changes for the Aurora release. Added introduction page which gives an overview of Project Aurora repositories. The Truffle migration still fails on me (when using Pet Shop example, I couldn't figure out in the remaining time why).

File: docs/develop/evm/introduction.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+[Project Aurora](https://aurora.dev/) delivers an Ethereum Virtual Machine
+(EVM) on the NEAR protocol. It is comprised of the following components:
+
+| Name                                                                                    | Description                            |
+| ----------------------------------------------------------------------------------------| -------------------------------------- |
+| **[Aurora Engine](https://github.com/aurora-is-near/aurora-engine)**                    | Implements an EVM on the NEAR protocol |
+| **[Aurora CLI](https://github.com/aurora-is-near/aurora-cli)**                          | Command-Line Interface (CLI) for the Aurora Engine |
+| **[Aurora Engine Client for JS](https://github.com/aurora-is-near/aurora.js)**          | JavaScript Client for the Aurora Engine |
+| **[NEAR Protocol Web3 provider](https://github.com/aurora-is-near/near-web3-provider)** | Web3 provider to connect Ethereum frontend or Truffle to NEAR Protocol
+| **[Aurora Relayer](https://github.com/aurora-is-near/aurora-relayer)** | A JSON-RPC server compatible with Ethereum's Web3 API for NEAR's EVM environment
```
Comment (artob) [inline docs/develop/evm/introduction.md:16]: Let's move the Relayer to precede the Web3 provider

File: docs/develop/evm/introduction.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+[Project Aurora](https://aurora.dev/) delivers an Ethereum Virtual Machine
+(EVM) on the NEAR protocol. It is comprised of the following components:
+
+| Name                                                                                    | Description                            |
+| ----------------------------------------------------------------------------------------| -------------------------------------- |
+| **[Aurora Engine](https://github.com/aurora-is-near/aurora-engine)**                    | Implements an EVM on the NEAR protocol |
+| **[Aurora CLI](https://github.com/aurora-is-near/aurora-cli)**                          | Command-Line Interface (CLI) for the Aurora Engine |
+| **[Aurora Engine Client for JS](https://github.com/aurora-is-near/aurora.js)**          | JavaScript Client for the Aurora Engine |
```
Comment (artob) [inline docs/develop/evm/introduction.md:14]: Let's use consistent title casing in the Description column. Right now it's title case for e.g. _"JavaScript Client for the Aurora Engine"_ but non-title case for _"Web3 provider to connect Ethereum frontend or Truffle to NEAR Protocol"_

File: docs/develop/evm/introduction.md (hunk 1 above)
Comment (artob) [inline docs/develop/evm/introduction.md:16]: Let's take off "for NEAR's EVM environment"

File: docs/develop/evm/introduction.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+id: introduction
+title: Project Aurora Introduction
+sidebar_label: Introduction
+---
+
+[Project Aurora](https://aurora.dev/) delivers an Ethereum Virtual Machine
+(EVM) on the NEAR protocol. It is comprised of the following components:
```
Comment (artob) [inline docs/develop/evm/introduction.md:8]: Check the official capitalization of "NEAR Protocol" elsewhere in docs

**Comment by artob** [comment]
> Depublished a few pages which need changes for the Aurora release. Added introduction page which gives an overview of Project Aurora repositories.

The update looks good. I won't have a chance to review and test in full before today's meeting, but aiming to get this done and merged by Monday.

> The Truffle migration still fails on me (when using Pet Shop example, I couldn't figure out in the remaining time why).

Do all other still-published examples work, it's just the Pet Shop?

File: docs/develop/evm/introduction.md (hunk 1 above)
Comment (frankbraun) [reply docs/develop/evm/introduction.md:16]: Done. I also changed the order of the corresponding pages accordingly.

**Review by artob — APPROVED** [review]

**Pull request merged**